src/Illuminate/Console/Command.php

```diff
@@ -282,22 +282,38 @@ public function table(array $headers, array $rows, $style = 'default')
 	 * Write a string as information output.
 	 *
 	 * @param  string  $string
+	 * @param  bool  $newline 
 	 * @return void
 	 */
-	public function info($string)
+	public function info($string, $newline = true) 
```

Comment:
In my opinion, all the other functions need to be *highlighted* in some way during the execution of Artisan command. A carriage return could be a way to highlight some text during the command execution. 
It is true, however, that the `newline` parameter is optional. So, the same behaviour is reached and `newline` can be extended to all the other functions.